examples/notebooks/introduction_enlyze_python_sdk.ipynb

```diff
@@ -0,0 +1,2981 @@
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                                   uuid display_name genesis_date  \\\n",
+       "0  22a71d1e-8262-4c27-b5fb-66598027936a    W&H Varex   2022-03-18   \n",
+       "1  cc0d2dcb-564b-48cd-a342-71765a536058       Macchi   2021-01-15   \n",
+       "2  141e0927-62b3-4e76-8398-ad82d20f397f       Kiefel   2020-08-26   \n",
+       "\n",
+       "                                                site  \n",
+       "0  Site(_id=32, display_name='Köln', address='Hel...  \n",
+       "1  Site(_id=32, display_name='Köln', address='Hel...  \n",
+       "2  Site(_id=32, display_name='Köln', address='Hel...  "
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "cologne_appliances = enlyze.get_appliances(site=site)\n",
+    "models_to_dataframe(cologne_appliances)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c6a57053-edac-4fcc-b3df-795ff9441d42",
+   "metadata": {},
+   "source": [
+    "**Selecting an appliance by UUID**\n",
+    "\n",
+    "A more reliable way to select an appliance from the list is filtering by `UUID`. Attributes like the name may change over time, but the assigned `UUID` will stay unchanged."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "ec91bd4f-0819-46a1-8738-c09839315bd0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from uuid import UUID"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "d95c523e-78f9-4c68-ae0f-e1fc77cca9bb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "MACHINE_UUID = UUID('22a71d1e-8262-4c27-b5fb-66598027936a')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2daff658-0ed4-4564-8a9f-b182cbbc436a",
+   "metadata": {},
+   "source": [
+    "We can create a function, that returns the appliance, if it matches the UUID:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "c920fd8d-bc74-4c0b-ab78-d6f5651dcba1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_appliance_by_uuid(uuid: UUID, appliances: List[Appliance]):\n",
+    "    for appliance in appliances:\n",
+    "        if appliance.uuid == uuid:\n",
+    "            return appliance\n",
+    "            "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "e0507860-468a-4923-a28b-861361355b5e",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Appliance(uuid=UUID('22a71d1e-8262-4c27-b5fb-66598027936a'), display_name='W&H Varex', genesis_date=datetime.date(2022, 3, 18), site=Site(_id=32, display_name='Köln', address='Heliosstrasse 6a'))"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "get_appliance_by_uuid(MACHINE_UUID, cologne_appliances)"
```

Comment:
Not a fan of mixing `appliance` and `machine`. We should go with one or another, but not both. It's okay to explain the aliasing at the beginning, but the rest should be consistent. I know this is hard because the SDK uses `Appliance`, but in case you really want to go with `Machine` here instead, then at least call you utility functions that you introduce accordingly.